packages/devtools_app/pubspec.yaml

```diff
@@ -23,10 +23,11 @@ dependencies:
   async: ^2.0.0
   codicon: ^1.0.0
   collection: ^1.15.0
-  dds_service_extensions: ^1.3.2
-  devtools_shared: 2.26.0-dev.6
+  dap: ^1.0.0
```

Comment:
I'd make this `dap: 1.1.0`. There's a new `lazy` field on `VariablePresentationHint ` that may be useful :-)

Reply:
Thanks! Done